lib/index.js

```diff
@@ -72,7 +86,7 @@ AppsFlyer.prototype.track = function(track, fn) {
   var props = reject(track.properties({ revenue: 'af_revenue' }));
   payload.eventValue = isEmpty(props) ? '' : stringify(props);
 
-  var ios = contains(track.library(), 'ios');
+  var ios = deviceType === 'ios';
```

Comment:
No, this is checking equality, not assigning value. This would return a boolean.